src/Storage/FileDriver.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+
+namespace Northwestern\SysDev\DynamicForms\Storage;
+
+
+use Illuminate\Http\JsonResponse;
+
+class FileDriver implements StorageInterface
```

Comment:
Not sure what to do with this yet, but this is definitely some kinda code smell -- we've got an interface that's implemented on two classes, and one of them is ignoring 75% of the interface.

Is there a way to better utilize `StorageInterface`?

Reply:
I agree, while writing this I was thinking of if StorageInterface should only include FindObject for now, and remove the other methods (and move the current S3Driver code into the trait)

Reply:
Removing the other three method from the interface is a good solution, but you don't have to move the S3 code out to the controller.

Those methods can be part of the S3Driver. If the controller actions are specific to S3, then it's fine to verify you're working with an S3 driver and then call its non-interface methods.

My desire is for the Drivers to do a bit more in the future -- I want some degree of interaction with the framework's [Filesystem](https://laravel.com/docs/8.x/filesystem) stuff. So that'll probably be when we add back more generic stuff to the interface? but that's for the future...

(sorry, got distracted!)